repos.d/gentoo/guru.yaml

```diff
@@ -0,0 +1,37 @@
+###########################################################################
+# GURU
+###########################################################################
+- name: guru
```

Comment:
There's dedicated file and `name`/`desc` scheme for Gentoo overlays, see repos.d/gentoo/overlays.yaml, please follow these.

Reply:
Move to `gentoo/overlays.yaml`, if I understood correctly, issued like Pentoo overlay.